src/models/nebula19-1/assembly.md

```diff
@@ -0,0 +1,401 @@
+- 1x CPU cooler w/ fan (optional)
+    - Noctua `NH-U9S` cooler w/ `NF-A9 PWM` fan
+- 1x GPU brace-mounted side fan (optional)
+    - Be Quiet! Silent Wings 4 120mm (`BQ SIW4-12025-MF-PWM`)
+
+### Steps to unpack the included accessories:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case) and [remove the CPU shroud](#removing-the-cpu-shroud).
+2. Remove the accessory boxes from the chassis.
+
+![Accessory boxes](./img/accessory-boxes.webp)
+
+3. Open the accessory boxes as needed.
+
+## Installing/removing the GPU brace:
+
+The GPU brace provides a mounting point to help keep installed GPUs (or other PCI Express cards) in place while transporting the system. It can also be used to mount up to two extra intake fans (one, a `BQ SIW4-12025-MF-PWM`, is optional when purchasing the chassis; a second is not included.)
+
+If no optional accessories were ordered with the nebula19, then the GPU brace will ship preinstalled. If optional accessories were included in the order, then they will ship inside of the chassis, and the GPU brace will ship in a separate box. It can be installed after assembling the rest of the computer.
+
+![GPU brace](./img/gpu-brace.webp)
+
+The fan splitter on the GPU brace can be used for any side intake fans installed on the GPU brace as well as the bottom chassis intake fan.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to unpack and install the GPU brace:
+
+1. [Remove the top case](#removing-the-top-case) and [unpack the included accessories](#unpacking-the-included-accessories).
+2. Cut the tape of the GPU brace box and remove the GPU brace and screw bag from the box.
+
+![GPU brace unboxing](./img/gpu-brace-unboxing.webp)
+
+3. Screw the GPU brace into the system using the four M3 screws included with the GPU brace.
+    - The fan splitter board should be on the right side (with the dual headers facing outwards) and the rockets should point upwards.
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+### Steps to remove the GPU brace:
+
+1. [Remove the top case](#removing-the-top-case).
+2. Unscrew the four screws holding the GPU brace in place (two on the front of the case, two on the back.)
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+3. Pull the GPU brace out of the chassis.
+
+## Installing the motherboard:
+
+nebula19 supports the following standard motherboard sizes:
+
+- Mini-DTX
+- Mini-ITX
+
+Four standoffs and motherboard screws are included.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to install the motherboard:
+
+1. If they are installed, [remove the top case](#removing-the-top-case), [remove the CPU shroud](#removing-the-cpu-shroud) and [remove the GPU brace](#installingremoving-the-gpu-brace).
+2. Locate the standoffs and M3 screws from the brown screw and velcro box.
+
+![Standoffs and M3 screws](./img/standoffs-screws.webp)
+
+3. Insert the four standoffs into the holes in the chassis.
```

Comment:
I think I didn't use "screw" because I didn't want to imply a screwdriver was needed.

Reply:
Alright, I guess it works. I'm mainly worried about the standoffs not being tight enough and coming loose instead of the motherboard screws when the motherboard screws are removed, but this new type of standoff still makes this less of an issue than the old style of standoff. The standoff would just need to be removed from the motherboard and put back into the chassis.